This patch is for festival for Debian GNU/Linux, which 
replaces four debug write calls with one

Index: b/src/modules/UniSyn_diphone/UniSyn_diphone.cc
===================================================================
--- a/src/modules/UniSyn_diphone/UniSyn_diphone.cc
+++ b/src/modules/UniSyn_diphone/UniSyn_diphone.cc
@@ -123,10 +125,10 @@
     }
     else
     {
-	*cdebug << ":" << get_param_str("grouped",args,"") << ":" << endl;
-	*cdebug << "index grouped:" << d_index->grouped << endl;
-	*cdebug << "true:" << true << endl;
-	*cdebug << "false:" << false << endl;
+	*cdebug << ":" << get_param_str("grouped",args,"") << ":" << endl
+	        << "index grouped:" << d_index->grouped << endl
+	        << "true:" << true << endl
+	        << "false:" << false << endl;
 	
 	d_index->coef_dir = get_param_str("coef_dir",args,"");
 	d_index->sig_dir = get_param_str("sig_dir",args,"");
